teep.cddl

```diff
@@ -36,33 +36,35 @@ TEEP-TYPE-teep-error = 5
 TEEP-TYPE-teep-success = 6
 
 version = uint .size 4
-ext-info = uint
+ext-info = uint 
 
-data-item-requested = $data-item-requested .within uint
+; data items as bitmaps
+data-item-requested = $data-item-requested .within uint .size 8
 attestation = 1
 $data-item-requested /=  attestation
 trusted-apps = 2
 $data-item-requested /= trusted-apps
-extensions = 3
+extensions = 4
 $data-item-requested /= extensions
-suit-commands = 4
+suit-commands = 8
 $data-item-requested /= suit-commands
 
 query-request = [
   type: TEEP-TYPE-query-request,
   token: uint,
   options: {
-    ? cipher-suites => [ + suite ],
+    ? supported-cipher-suites => suite,
     ? nonce => bytes,
     ? version => [ + version ],
     ? oscp-data => bytes,
     * $$query-request-extensions
     * $$teep-option-extensions
   },
-  * data-item-requested,  ; why is this not a bitmap?
+  data-item-requested  
 ]
 
-suite = $TEEP-suite .within uint
+; ciphersuites as bitmaps
+suite = $TEEP-suite .within uint .size 8
```

Comment:
Maybe suite-selection. Also, I am not sure, if a bitmap is the best way to do this. How many options do we expect in the foreseeable future? And how many selected suites can be expected, typically?

In general, this should work. Again, just checking.